tests/unit/records/sources/test_fileobjs.py

```diff
@@ -99,15 +99,15 @@ def test_known_supported_records_formats(self):
         out = source.known_supported_records_formats()
         self.assertEqual(out, [mock_records_format])
 
-    def test_can_move_to_this_format_yes(self):
+    def test_can_move_to_format_yes(self):
```

Comment:
Also, `yes` should be `true` here as well, I think?

Reply:
✔️